modules/host/announce_test.go

```diff
@@ -45,7 +45,7 @@ func newAnnouncementFinder(cs modules.ConsensusSet) (*announcementFinder, error)
 	af := &announcementFinder{
 		cs: cs,
 	}
-	err := cs.ConsensusSetSubscribe(af, modules.ConsensusChangeBeginning)
+	err := cs.ConsensusSetSubscribe(af, modules.ConsensusChangeBeginning, nil)
```

Comment:
Didn't pass the channel here because I don't think that it really matters that much for those tests.